vpn/__init__.py

```diff
@@ -0,0 +1,70 @@
+"""VPN Extension
+
```

Comment:
this text will apear in the setings dialog. please use it only if it is beneficial for users. title is in the config and will be duplicated.